Doc/whatsnew/2.1.rst

```diff
@@ -27,7 +27,7 @@ of new releases, with a new release coming every 6 to 9 months. 2.1 is the first
 release to come out at this faster pace, with the first alpha appearing in
 January, 3 months after the final version of 2.0 was released.
 
-The final release of Python 2.1 was made on April 17, 2001.
+The final release of Python 2.1 was released on April 17, 2001.
```

Comment:
"release ... was released" doesn't scan terribly well. I don't think this line needs to change.

A

Reply:
This change was made to retain consistency throughout the entire documentation among different versions. TBH, it is definitely not made on April 17, 2001 rather released.

Reply:
Changing the text of a 21-year old whatsnew document purely for consistency seems iffy, I remain -1 on it.

A

Reply:
Agree, this change is unnecessary